ranger/core/fm.py

```diff
@@ -160,7 +161,7 @@ def sxiv_workaround_hook(command):
                         "$@" in command:
                     new_command = None
 
-                    if command[0:5] == 'sxiv ':
+                    if command[0:5] == 'sxiv ' or command[0:6] == 'nsxiv':
```

Comment:
This PR fixes the issue for me - just a small typo needs to be fixed
```suggestion
                    if command[0:5] == 'sxiv ' or command[0:6] == 'nsxiv ':
```

Reply:
Thanks!